homeassistant/components/demo/cover.py

```diff
@@ -205,19 +205,19 @@ async def async_open_cover_tilt(self, **kwargs: Any) -> None:
         self._listen_cover_tilt()
         self._requested_closing_tilt = False
 
-    async def async_set_cover_position(self, **kwargs):
+    async def async_set_cover_position(self, **kwargs: Any) -> None:
         """Move the cover to a specific position."""
-        position = kwargs.get(ATTR_POSITION)
+        position: int = kwargs[ATTR_POSITION]
```

Comment:
The `position` argument should always be present, and round doesn't like nullables.